packages/components/src/resizable-box/resize-tooltip/utils.ts

```diff
@@ -60,7 +60,7 @@ export function useResizeLabel( {
 	 * This custom hook uses injects an iFrame into the element, allowing it
 	 * to tap into the onResize (window) callback events.
 	 */
-	const [ resizeListener, sizes ] = useResizeAware();
+	const [ resizeListener, sizes ] = useResizeObserver();
```

Comment:
Is the comment above still true?

Reply:
I guess it's a div and not an iframe anymore.